sdk/desktopvirtualization/Azure.ResourceManager.DesktopVirtualization/api/Azure.ResourceManager.DesktopVirtualization.netstandard2.0.cs

```diff
@@ -807,16 +814,37 @@ public partial class RegistrationInfoPatch
         public static bool operator !=(Azure.ResourceManager.DesktopVirtualization.Models.RemoteApplicationType left, Azure.ResourceManager.DesktopVirtualization.Models.RemoteApplicationType right) { throw null; }
         public override string ToString() { throw null; }
     }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct ResourceIdentityType : System.IEquatable<Azure.ResourceManager.DesktopVirtualization.Models.ResourceIdentityType>
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public ResourceIdentityType(string value) { throw null; }
+        public static Azure.ResourceManager.DesktopVirtualization.Models.ResourceIdentityType SystemAssigned { get { throw null; } }
+        public bool Equals(Azure.ResourceManager.DesktopVirtualization.Models.ResourceIdentityType other) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override bool Equals(object obj) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override int GetHashCode() { throw null; }
+        public static bool operator ==(Azure.ResourceManager.DesktopVirtualization.Models.ResourceIdentityType left, Azure.ResourceManager.DesktopVirtualization.Models.ResourceIdentityType right) { throw null; }
+        public static implicit operator Azure.ResourceManager.DesktopVirtualization.Models.ResourceIdentityType (string value) { throw null; }
+        public static bool operator !=(Azure.ResourceManager.DesktopVirtualization.Models.ResourceIdentityType left, Azure.ResourceManager.DesktopVirtualization.Models.ResourceIdentityType right) { throw null; }
+        public override string ToString() { throw null; }
+    }
     public partial class ResourceModelWithAllowedPropertySet : Azure.ResourceManager.Models.TrackedResourceData
     {
         public ResourceModelWithAllowedPropertySet(Azure.Core.AzureLocation location) : base (default(Azure.Core.AzureLocation)) { }
         public string Etag { get { throw null; } }
-        public Azure.ResourceManager.Models.SystemAssignedServiceIdentity Identity { get { throw null; } set { } }
+        public Azure.ResourceManager.DesktopVirtualization.Models.ResourceModelWithAllowedPropertySetIdentity Identity { get { throw null; } set { } }
```

Comment:
replaced by `Models.SystemAssignedServiceIdentity` is better

Reply:
[This comment](https://github.com/Azure/azure-sdk-for-net/pull/29073#discussion_r896476945) has detailed reasons about why this happens.
Current implementation cannot replace this automatically, and the behavior to replace this type in previous generation is wrong: because the Type inside this does not have value "None". It might indicate that the service may not accept the value `None` that this is not documented in the corresponding swagger
We need to validate this before we make any change on this. And if you want to the result of getting replaced by the common-types, we need to fix the swagger by directive or ask the service team to change their swaggers.

Reply:
yes, it's ok now as this RP is still beta, please log a new issue to tray the post efforts